doc/source/reference/arrays.dtypes.rst

```diff
@@ -178,7 +178,7 @@ Built-in Python types
     :class:`bool`     :class:`bool\_`
     :class:`float`    :class:`float\_`
     :class:`complex`  :class:`cfloat`
-    :class:`str`      :class:`string`
+    :class:`str`      :class:`bytes` (Python2) or :class:`unicode\_` (Python3)
```

Comment:
I agree it should be kept so people can look up what it means. I have added a note in the section.